Misc/NEWS.d/next/Library/2020-09-08-23-41-29.bpo-41747.M6wLKv.rst

```diff
@@ -0,0 +1,2 @@
+Ensure all methods from a :func:`dataclasses.dataclass` uses the
+``__qualname__`` of the class they belong. Patch by Batuhan Taskaya.
```

Comment:
```suggestion
Ensure all methods belonging to a :func:`dataclasses.dataclass` objects now have
the propper ``__qualname__`` attribute referring to the class they belong. Patch by Batuhan Taskaya.
```

Reply:
ups, there is a typo (`propper`). Will commit individually!

Reply:
Thanks! Comitted